diff -Nru mawk-1.3.3/examples/ct_length.awk mawk-1.3.3.new/examples/ct_length.awk
--- mawk-1.3.3/examples/ct_length.awk	Sat Jul  3 20:58:46 1993
+++ mawk-1.3.3.new/examples/ct_length.awk	Wed Feb  9 07:56:44 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/mawk -f
+#!/usr/bin/mawk -f
 
 #  ct_length.awk
 #
diff -Nru mawk-1.3.3/examples/hcal mawk-1.3.3.new/examples/hcal
--- mawk-1.3.3/examples/hcal	Sun Aug 25 20:45:47 1996
+++ mawk-1.3.3.new/examples/hcal	Wed Feb  9 07:56:55 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/mawk -We
+#!/usr/bin/mawk -We
 # edit the above to be the full pathname of 'mawk'
 # @(#) hcal - v01.00.02 - Tue Feb 27 21:21:21 EST 1996
 # @(#) prints a 3-month (highlighted) calendar centered on the target month
